ui/cryptomaterial/util.go

```diff
@@ -139,20 +139,26 @@ func AnyKeyWithOptionalModifier(modifier key.Modifiers, keys ...string) key.Set
 	return key.Set(keysWithModifier)
 }
 
-func UniformPadding(gtx layout.Context, body layout.Widget) layout.Dimensions {
+func UniformPadding(gtx layout.Context, body layout.Widget, isMobileView ...bool) layout.Dimensions {
+	_isMobileView := len(isMobileView) > 0 && isMobileView[0]
```

Comment:
```suggestion
	isMobileView := len(isMobileView) > 0 && isMobileView[0]
```

Reply:
isMobileView is user on param of function